pkgs/games/multimc/default.nix

```
@@ -33,7 +33,16 @@ in mkDerivation rec {
       Allows you to have multiple, separate instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple interface.
     '';
     platforms = platforms.linux;
-    license = licenses.lgpl21Plus;
+    license = with licenses; [
+      # Per https://github.com/MultiMC/MultiMC5/blob/develop/COPYING.md
```

```suggestion
      # Per COPYING.md in source tree
```
could be the comment